src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java

```diff
@@ -117,6 +117,7 @@ private KafkaConfig createConfig()
         config.put(KafkaConfig.DeleteTopicEnableProp(), "false");
         config.put(KafkaConfig.ControlledShutdownRetryBackoffMsProp(), "100");
         config.put(KafkaConfig.LogCleanerDedupeBufferSizeProp(), "2097152");
+        config.put(KafkaConfig.OffsetsTopicReplicationFactorProp(), (short) 1);
```

Comment:
I haven't reviewed the provisions of this library in detail, but is there another place where we can conveniently set this replication factor to a non-1 value by default?  I recall a problem where it defaulted to 1, so we didn't have any redundancy (e.g., some of the chat2 topics on chimera kafka).